UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift

```diff
@@ -94,13 +95,34 @@ class BTPayPalClient_Tests: XCTestCase {
         XCTAssertEqual(lastPostParameters["cancel_url"] as? String, "sdk.ios.braintree://onetouch/v1/cancel")
     }
 
-    func testTokenizePayPalAccount_whenPaymentResourceCreationFails_callsBackWithError() {
-        mockAPIClient.cannedResponseError = BTPayPalError.httpResponseMissingUserInfoJSON as NSError
+    func testTokenizePayPalAccount_whenPoseRequstContainsError_callsBackWithError() {
```

Comment:
Looks like a small typo in the test name. Is this supposed to be `whenPostRequestContainsError`?

Reply:
🤦